causallearn/search/ConstraintBased/FCI.py

```diff
@@ -82,8 +82,7 @@ def existOnePathWithPossibleParents(self, previous, node_w: Node, node_x: Node,
                 continue
 
             if self.existsSemidirectedPath(node_r, node_x, graph) or self.existsSemidirectedPath(node_r, node_b, graph):
-                if self.existOnePathWithPossibleParents(previous, node_r, node_x, node_b, graph):
-                    return True
+                return True
```

Comment:
Yes, this will cause "RecursionError: maximum recursion depth exceeded while calling a Python object". And I compared the [code](https://github.com/cmu-phil/tetrad/blob/70b0506af635d4f8906b3e29124bb5d01343768b/tetrad-lib/src/main/java/edu/cmu/tetrad/graph/GraphUtils.java#L4768) in Tetrad and it seems that these two lines of code are not needed.